pkg/services/pluginsintegration/plugininstaller/service.go

```diff
@@ -101,7 +101,10 @@ func (s *Service) installPluginsWithTimeout(pluginsToInstall []setting.InstallPl
 	}
 }
 
-func (s *Service) shouldUpdate(ctx context.Context, pluginID, currentVersion string) bool {
+func (s *Service) shouldUpdate(ctx context.Context, pluginID, currentVersion string, pluginURL string) bool {
+	if pluginURL != "" {
```

Comment:
@andresmgot @wbrowne With this change(URL set and no version specified) we are allowing downgrade.